docs/how-to/setting_up_a_survey.md

```diff
@@ -0,0 +1,70 @@
+This page provides an overview of how to utilize surveys in your OCS chatbot.
+
+### What are Surveys on Open Chat Studio?
+
+The Surveys feature allows chatbot makers to give users a link to a Google form (or any other link to a survey), both at the start and end of an OCS chatbot web session. 
```

Comment:
I think it would be good to add a 'note' admonition to highlight that surveys only work on the web